### PR TITLE
Azure Stack Fabric.Admin Api update for GPU support

### PR DESCRIPTION
**Pull request by LingyunSu**
Azure Stack Fabric.Admin Api update for GPU support

### Contribution checklist:
- [X] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [X] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [X] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by chiragg4u** [comment]
Thank you for copying the existing version into new folder for first commit. Changes looks good to me marking it ARMSignedOff.

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnitNode.json [hunk 1]
```diff
@@ -315,19 +315,19 @@
         "operationId": "ScaleUnitNodes_List",
         "parameters": [
           {
-            "$ref": "Fabric.json#/parameters/SubscriptionIdParameter"
+            "$ref": "../2016-05-01/Fabric.json#/parameters/SubscriptionIdParameter"
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnitNode.json:318]: is it recommended to make a copy of Fabric.json to 2020-10-01 folder ? @allenjzhang

**Review by bganapa — APPROVED** [review]

**Comment by bganapa** [comment]
second commit for easy review https://github.com/Azure/azure-rest-api-specs/pull/11092/commits/9ceaa94e815bd7332654711bcde84ec27e36d044

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnitNode.json (hunk 1 above)
Comment (LingyunSu) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnitNode.json:318]: Direct copying this file, will cause a version problem. Other models are still in old version.  I just followed the existing solution of referencing the old model, like here:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2018-10-01/Drive.json

**Review by allenjzhang — CHANGES_REQUESTED** [review]
Please fix checks failures. The swaggers look fine. Once checks are green, it can be merged. Thx

**Comment by bganapa** [comment]
@allenjzhang 
The following error is an issue with the linting tool.  RP Name is already in PascalCasing, i believe since we have .Admin part in the RP name, the linter is flagging it as an issue. 

Resource provider naming must follow the pascal case. Path: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Fabric.Admin/fabricLocations/{location}/scaleUnits/{scaleUnit}/setGpuPartitionSize'
  at specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnit.json:62

https://dev.azure.com/azure-sdk/public/_build/results?buildId=575227&view=logs&j=688669d0-441c-57c3-cf6d-f89a22ccfa5d&t=923b5a90-e462-5994-c166-86509c6bf5f2&l=3754

**Comment by allenjzhang** [comment]
@akning-ms , could you help to see if this is caused by our validation logic can't handles provider name with two dots? 


R3020 - PathResourceProviderNamePascalCase
Resource provider naming must follow the pascal case. Path: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/**Microsoft.Fabric.Admin**/fabricLocations/{location}/scaleUnits/{scaleUnit}/setGpuPartitionSize'
  at specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnit.json:62

**Review by allenjzhang — APPROVED** [review]

**Comment by akning-ms** [comment]
> @akning-ms , could you help to see if this is caused by our validation logic can't handles provider name with two dots?
> 
> R3020 - PathResourceProviderNamePascalCase
> Resource provider naming must follow the pascal case. Path: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/**Microsoft.Fabric.Admin**/fabricLocations/{location}/scaleUnits/{scaleUnit}/setGpuPartitionSize'
> at specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnit.json:62

Yes.  This case already supported in new Lint version which will enable soon. @jianyexi for awareness

**Pull request merged**